phpcs.xml.dist

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<ruleset name="Custom ruleset for Liveblog">
+
+	<rule ref="WordPress-Extra"/>
+	<rule ref="WordPressVIPMinimum"/>
+
+	<arg value="sp"/>
+	<arg name="colors"/>
+	<arg name="basepath" value="."/>
+	<arg name="parallel" value="50"/>
+	<arg name="extensions" value="php"/>
+	<file>.</file>
+
```

Comment:
Would like to see text domain checks enabled, and prefixes, and some link references.

See https://github.com/GaryJones/plugin-boilerplate/blob/master/.phpcs.xml.dist, and consider copying this, and amending it with the liveblog bits (no need to include Neutron).

PHPCompatibilityWP standard would also then need to be added to `composer.json` and Travis config.

Reply:
Let's leave this for a separate change